high-level.lisp

```diff
@@ -817,6 +817,30 @@ with upper left block with dimension P-by-Q. Returns the intermediate representa
         (magicl.lapack-cffi::%zgetri rows a lda ipiv work lwork info)
         (values (make-matrix :rows rows :cols cols :data a))))))
 
+(defun dagger (m)
```

Comment:
This is already implemented as `conjugate-transpose` on line 353 of this file. I'm happy for this to be a synonym, though; it's a lot more pleasant to write.